src/java/com/zimbra/qa/selenium/projects/admin/pages/PageZextrasHSM.java

```diff
@@ -27,23 +27,23 @@
 public class PageZextrasHSM extends AbsTab {
 
 	public static class Locators {
-		public static final String NETWORK_MODULE_NG_ICON = "css=td[id^='zti__AppAdmin__Home__ZeXtras_'] div[class='ImgZeXtras']";
-		public static final String HSM_TAB = "css=td[id^='zti__AppAdmin__ZeXtras__ZxPowerstore_']:contains('HSM')";
+		public static final String NETWORK_MODULE_NG_ICON = "css=td[id^='zti__AppAdmin__Home__Zimbra Network'] div[class='ImgZeXtras']";
+		public static final String HSM_TAB = "css=td[id^='zti__AppAdmin__Home__Zimbra Network']:contains('HSM')";
 		public static final String CLOSE_BUTTON = "css=td[id^='zb__ZaCurrentAppBar__CLOSE_']:contains('Close')";
 		public static final String SAVE_BUTTON = "css=td[id^='zb__ZaCurrentAppBar__SAVE_']:contains('Save')";
 		public static final String APPLY_HSM_POLICY_NOW_BUTTON = "css=td[class='ZWidgetTitle']:contains('Apply HSM Policy Now')";
-		public static final String SECONDARY_VOLUME_ADD_BUTTON = "css=div[id='ztabv__ZxPowerstore_zawiz_top_grouper_2'] td[class='ZWidgetTitle']:contains('Add')";
+		public static final String SECONDARY_VOLUME_ADD_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_10'] td[class='ZWidgetTitle']:contains('Add')";
 		public static final String SECONDARY_VOLUME_EDIT_BUTTON = "css=div[id='ztabv__ZxPowerstore_zawiz_top_grouper_2'] td[class='ZWidgetTitle']:contains('Edit')";
 		public static final String SECONDARY_VOLUME_DELETE_BUTTON = "css=div[id='ztabv__ZxPowerstore_zawiz_top_grouper_2'] td[class='ZWidgetTitle']:contains('Delete')";
-		public static final String HSM_POLICY_ADD_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_14'] td[class='ZWidgetTitle']:contains('Add')";
-		public static final String HSM_POLICY_EDIT_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_14'] td[class='ZWidgetTitle']:contains('Edit')";
-		public static final String HSM_POLICY_DELETE_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_14'] td[class='ZWidgetTitle']:contains('Delete')";
-		public static final String lastPolicyInList = "css=div.hsmList div[class='DwtListView-Rows'] div:last-child";
+		public static final String HSM_POLICY_ADD_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_16'] td[class='ZWidgetTitle']:contains('Add')";
+		public static final String HSM_POLICY_EDIT_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_16'] td[class='ZWidgetTitle']:contains('Edit')";
+		public static final String HSM_POLICY_DELETE_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_16'] td[class='ZWidgetTitle']:contains('Delete')";
```

Comment:
Need to update the locators.
div id value contains number at the end ('ztabv__ZxPowerstore_group_16') which will changed on every login

Reply:
I have verifiied 'ztabv__ZxPowerstore_group_16' locator. It is constant and not changing changing on every login.